bigtable/filter.go

```diff
@@ -144,6 +144,18 @@ func (lnf latestNFilter) proto() *btpb.RowFilter {
 	return &btpb.RowFilter{Filter: &btpb.RowFilter_CellsPerColumnLimitFilter{CellsPerColumnLimitFilter: int32(lnf)}}
 }
 
+// ApplyLabelFilter returns a filter that applies the
+// given label to all cells in the output row.
+func ApplyLabelFilter(label string) Filter { return applyLabelFilter(label) }
+
+type applyLabelFilter string
+
+func (alf applyLabelFilter) String() string { return fmt.Sprintf("apply_label(%s)", string(alf)) }
+
+func (alf applyLabelFilter) proto() *btpb.RowFilter {
+	return &btpb.RowFilter{Filter: &btpb.RowFilter_ApplyLabelTransformer{ApplyLabelTransformer: string(alf)}}
+}
+
```

Comment:
Adding a labels apply filter given [the limitation](https://github.com/googleapis/go-genproto/blob/053ba62fc06f7e7fd79767598c2e73fbae71e745/googleapis/bigtable/v2/data.pb.go#L1262-L1263)